rclcpp/include/rclcpp/experimental/timers_manager.hpp

```diff
@@ -125,9 +125,14 @@ class TimersManager
   /**
    * @brief Starts a thread that takes care of executing the timers stored in this object.
    * Function will throw an error if the timers thread was already running.
+   *
+   * @param exception_handler if valid, the execution of the timer will be done in a try catch block,
```

Comment:
I'll leave this `@`, because this file uses that style, but the whole file should be updated (it mostly got a review pass for being in the `experimental` namespace.